File: basicProgramming/c++/ifEsle/p3.cpp
#include <iostream>
using namespace std;

int main()
{
    // !Write a program to find the maximum between two numbers.
    int num1, num2;
    cin >> num1;
    cin >> num2;
    if (num1 > num2)
    {
        cout << "Num1 is greater" << num1 << endl;
    }

    else
    {
        cout << "Num2 is greater " << num2 << endl;
    }

    return 0;
}

File: basicProgramming/dart/ifElse/p3.dart
//! Write a program to find the maximum between two numbers.

import 'dart:io';

void main() {
  int num1, num2, num3;

  num1 = int.parse(stdin.readLineSync()!);
  num2 = int.parse(stdin.readLineSync()!);
  num3 = int.parse(stdin.readLineSync()!);

  if (num1 > num2 && num1 > num3) {
    print("num1 is greater ");
  } else if (num2 > num1 && num2 > num3) {
    print("num2 is greater ");
  } else {
    print("Num3 is greater ");
  }
}
